### DISCUSSION TITLE
Which module are you using?

### DISCUSSION THREAD

**Discussion by tommyme** [tommyme/vscode_bookmark_x#31] [Polls]
bookmark-x contains 3 modules:

- bmx -- bookmark module
- quick run -- task & debug runtime manager
- referLink -- some tricky commands

**Comment by soda92** [comment]
well I tried the referlink module, the path commands are great!

I'm always struggled with paths in vscode, and i've done countless manual converations in config files...

**Reply by tommyme** [reply]
Very cool, especially since you've tried all three modules. I hope BMX will make your work even more enjoyable. 😊
